### PR TITLE
Update ArabicDictionary.php & EnglishDictionary.php

### PR DESCRIPTION
**Pull request by sersy**
Add AED & EGP Currency with Arabic and English dictionary

### CODE REVIEW

File: src/Language/English/EnglishDictionary.php [hunk 1]
```diff
@@ -66,6 +66,7 @@ class EnglishDictionary implements Dictionary
         'LVL' => [['lat'], ['sentim']],
         'LYD' => [['dinar'], ['cent']],
         'MAD' => [['dirham'], ['cent']],
+        'AED' => [['Dirham'], ['Fils']],
```
Comment (kwn) [inline src/Language/English/EnglishDictionary.php:69]: Could you please add the currency in the alphabetical order (top of the array).

**Review by kwn — CHANGES_REQUESTED** [review]
Thank you. Could you please add a few unit tests?

File: src/Language/English/EnglishDictionary.php (hunk 1 above)
Comment (sersy) [reply src/Language/English/EnglishDictionary.php]: this is done now thx

**Comment by kwn** [comment]
Hi @sersy 

Could you please add a few unit tests? I'd like to make sure the functionality is tested properly before merging it.

**Pull request merged**